core/templates/dev/head/css/oppia.css

```diff
@@ -3378,7 +3378,7 @@ md-card.preview-conversation-skin-supplemental-card {
   cursor: pointer;
   display: inline-block;
   height: 258px;
-  margin: 12px 4px;
+  margin: 12px 7px;
```

Comment:
Maybe explain that this is needed so that the exploration and collection summary tiles have the same width?

That said, isn't a collection tile made up of three exploration tiles? Does this make its width expand as well?